bubble-sort.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const array = [7, 5, 6, 4, 3, 10, 1, 8, 2, 9];
+
+function bubbleSort(array) {
+  let swapped;
+  do {
+    swapped = false;
+    for (let i = 0; i < array.length; i++) {
+      if (array[i] && array[i + 1] && array[i] > array[i + 1]) {
+        swap(array, i, i + 1);
```

Comment:
this `swap` function is not defined anywhere, so it breaks your code, add the swap function somewhere in this file and it should work properly.